sdk/storage/Azure.Storage.Blobs/src/PartitionedUploader.cs

```diff
@@ -243,7 +243,9 @@ internal class PartitionedUploader
                 IAsyncEnumerator<ChunkedStream> enumerator =
                     PartitionedUploadExtensions.GetBlocksAsync(content, blockSize, async: false, _arrayPool, cancellationToken)
                     .GetAsyncEnumerator(cancellationToken);
+#pragma warning disable AZC0107
```

Comment:
Why did this fire?

Reply:
There is no synchronous version of `MoveNextAsync`, `async` parameter is passed to `GetBlocksAsync` that creates `IAsyncEnumerable<ChunkedStream>` which works synchronously.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/10161/